### DISCUSSION TITLE
How do I donate to this project?

### DISCUSSION THREAD

**Discussion by epabst** [chris1111/Wireless-USB-Big-Sur-Adapter#138] [General]
I am incredibly grateful for this project.  It asked me to donate, but I hadn't tested it yet, and now I don't know where I can donate.  I'd prefer Venmo if that is an option.

**Comment by chris1111** [comment]
@epabst 
Its here  [paypal](https://www.paypal.com/donate/?cmd=_donations&business=V9C6559FRX5BC&item_name=Support&currency_code=CAD&source=url)

Thank you for support

**Comment by chris1111** [comment]
Donation receive thanks
